challenges/3-area-trapezio/c/Danteborges/README.md

```diff
@@ -0,0 +1,22 @@
+# Submissão de Exercicio
+
+**Exercicio:** 3 - Área de um trapézio
+
+**Nickname:** DanteBorges
+
+**Nível Técnico:** Estagiário
+
+**Twitter**: https://twitter.com/Borges_Dante
+
+**Dificuldade de Resolução:** Baixa
+
+**Comentários:**
```

Comment:
Se você não for implementar comentários, por favor, remova a linha.

Reply:
removido